templates/web/tfl/email_sent.html

```diff
@@ -6,9 +6,17 @@ <h1>[% loc("Nearly done! Now check your email&hellip;") %]</h1>
 
     <p>
     [% IF email_type == 'problem' %]
-        We’ve sent you a confirmation email. Click the link to publish your problem.
+        [% IF non_public %]
+            [% loc("We've sent you a confirmation email. Click the link to submit your problem.") %]
```

Comment:
These don't need marking for translating (though I guess doesn't matter if you do, most cobrand directories are not looked at for finding strings).